README.md

```diff
@@ -18,11 +18,43 @@ jobs:
         anaconda_nightly_upload_token: ${{secrets.UPLOAD_TOKEN}}
 ```
 
-Note that we recommend pinning the action against a specific SHA
+> [!IMPORTANT]
+> Note that we recommend pinning the action against a specific SHA
 (rather than a tag), to guard against the unlikely event of upstream
 being compromised.
 
-## Updating the action
+# Removing old nightly builds
```

Comment:
```suggestion
## Removing old nightly builds
```

Reply:
Presuming `#` is kept fot title?

Reply:
Oh, I was keeping this because this repository would support two major actions. I'll keep just one title for the README.

Reply:
Addressed in 556e9a6bca0f2d1ff406804ed59829400c6a9fbe.